packages/flutter_tools/test/general.shard/build_system/targets/localizations_test.dart

```diff
@@ -0,0 +1,118 @@
+// Copyright 2014 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:file/memory.dart';
+import 'package:flutter_tools/src/base/file_system.dart';
+import 'package:flutter_tools/src/base/logger.dart';
+import 'package:flutter_tools/src/build_system/depfile.dart';
+import 'package:flutter_tools/src/build_system/targets/localizations.dart';
+
+import '../../../src/common.dart';
+import '../../../src/context.dart';
+
+void main() {
+  testWithoutContext('generateLocalizations forwards arguments correctly', () async {
+    final FileSystem fileSystem = MemoryFileSystem.test();
+    final Logger logger = BufferLogger.test();
+    final FakeProcessManager processManager = FakeProcessManager.list(<FakeCommand>[
```

Comment:
If I understand how `FakeProcessManager` works, the list of `FakeCommand`s here is the list of commands that have to be run in this order, right?

Reply:
Yes, any other commands would otherwise fail